mycroft/skills/phillips_hue/vocab/en-us/TurnOnKeyword.voc

```diff
@@ -0,0 +1,4 @@
+turn my lights on
+turn on my lights
+turn on the lights
+turn the lights on
```

Comment:
Similar what I suggested before, use a `LightKeyword` and `OnKeyword` here to clean up the vocab. The [Volume skill](https://github.com/MycroftAI/mycroft-core/tree/master/mycroft/skills/volume) has good examples.